# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1299173344 28800
#      Thu Mar 03 09:29:04 2011 -0800
# Node ID 1536ed54621982023931693a0e7fb492c6102dd1
# Parent  245b5efb07c2b1b9cf395441d6291ea2fd2d36cc
Fix bug #32443 preventing 'pkg install' on Windows platforms.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-03-03  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/movefile.m: Fix copy&paste bug affecting Windows
+	platform.  (bug #32443)
+
 2010-03-02  Rik  <octave@nomad.inbox5.com>
 
 	* statistics/distributions/unidcdf.m,
 	statistics/distributions/unidinv.m, statistics/distributions/unidpdf.m,
 	statistics/distributions/unidrnd.m: Correctly refer to distribution as
 	discrete uniform in docstring.
 
 2011-03-02  Konstantinos Poulios  <logari81@gmail.com>
diff --git a/scripts/miscellaneous/movefile.m b/scripts/miscellaneous/movefile.m
--- a/scripts/miscellaneous/movefile.m
+++ b/scripts/miscellaneous/movefile.m
@@ -105,17 +105,17 @@ function [status, msg, msgid] = movefile
         [err, msg] = system (sprintf ("%s %s \"%s\"", cmd, p1, p2));
         if (err < 0)
           status = false;
           msgid = "movefile";
         endif
       endwhile
     else
       if (ispc () && ! isunix ()
-          && ! isempty (file_in_path (getenv_path ("PATH"), "cp.exe")))
+          && ! isempty (file_in_path (getenv ("PATH"), "cp.exe")))
         p1 = strrep (p1, "\\", "/");
         p2 = strrep (p2, "\\", "/");
       endif
 
       ## Move the file(s).
       [err, msg] = system (sprintf ("%s %s \"%s\"", cmd, p1, p2));
       if (err < 0)
         status = false;
